### PR TITLE
Redis test failiures

### PR DESCRIPTION
**Pull request by Amir-61**
Hey Simon,

Please review

Thanks alot!

### CODE REVIEW

**Comment by Amir-61** [comment]
@superkhau Please review

Thanks!

File: test/datatype.test.js [hunk 1]
```diff
@@ -48,19 +48,22 @@ describe('datatypes', function () {
...
       str: 'hello', date: d, num: '3', bool: 1, list: ['test'], arr: [1, 'str']
     }, function (err, m) {
       should.not.exists(err);
       should.exist(m && m.id);
-      m.str.should.be.type('string');
-      m.num.should.be.type('number');
-      m.bool.should.be.type('boolean');
+      should(m.str).be.type('string');
+      should(m.num).be.type('number');
+      should(m.bool).be.type('boolean');
```
Comment (superkhau) [inline test/datatype.test.js]: Please upgrade should ASAP so you can use the other syntax.

**Comment by superkhau** [comment]
LGTM

File: test/datatype.test.js [hunk 2]
```diff
@@ -187,21 +186,21 @@ describe('datatypes', function () {
     });
 
     it('should set missing optional properties to null', function(done) {
-      var EXPECTED = { desc: null, stars: null };
```
Comment (superkhau) [inline test/datatype.test.js]: This needs to stay.

File: test/datatype.test.js [hunk 3]
```diff
@@ -187,21 +186,21 @@ describe('datatypes', function () {
...
-          found.should.have.properties(EXPECTED);
+          created.should.have.property('desc', null);        
+          created.should.have.property('stars', null);
           done();
         });
       });
     });
 
-    it('should convert property value undefined to null', function(done) {
-      var EXPECTED = { desc: null, extra: null };
```
Comment (superkhau) [inline test/datatype.test.js]: This also needs to stay.

**Pull request merged**